xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/displayers/BaseDisplayer.vue

```diff
@@ -91,55 +95,127 @@ export default {
   // Add the displayerMixin to get access to all the displayers methods and computed properties inside this component
   mixins: [displayerMixin],
 
+  components: {
+    XWikiLoader,
+  },
 
   props: {
     viewOnly: {
       type: Boolean,
       default: false,
     },
-  },
-
-  data () {
-    return {
-      // Whether the displayer is in view or edit mode
-      isView: true,
-    };
+    isView: {
+      type: Boolean,
+      default: true,
+    },
+    isLoading: {
+      type: Boolean,
+      default: false
+    }
   },
 
   computed: {
-    isEditable () {
-      return this.logic.isEditable({
+    // Checks if the displayed is allowed to be edited and if the livedata is in a state where this displayer can be
+    // edited.
+    isEditable() {
+      const editable = this.logic.isEditable({
         entry: this.entry,
         propertyId: this.propertyId,
       });
+      // Checks that no other cell is already editing.
+      const noOtherEditing = this.editBus.isEditable()
+      return editable && noOtherEditing;
+    },
+    // The base value uses the value provided in the props the initial value of the form input.
+    // Once the form is edited, `this.editedValue` is defined and is used instead. 
```

Comment:
Could you improve this comment. It's not clear at all **why** you need this ``baseValue``.